openapi/resource-server.yaml

```diff
@@ -167,7 +167,6 @@ paths:
                       assetScale: 2
```

Comment:
should we bump the version of the spec? 
There's a few different ways to do this, Stripe has an interesting approach where they use dates instead of version numbers when breaking changes are released:
- https://stripe.com/docs/api/versioning
- https://news.ycombinator.com/item?id=13708927
- https://keygen.sh/blog/breaking-things-without-breaking-things/

Reply:
Probably should be a wider discussion, but I think we do need to version this change somehow

Reply:
Yes, definitely agree since this is a breaking change. I'm reading those sources and some others and think we should talk about this further.